File: frontend/src/pages/convertFont.js
const fs = require('fs');
const font = fs.readFileSync('./src/fonts/NanumGothic-Regular.ttf');
const b64 = font.toString('base64');
fs.writeFileSync('./src/fonts/NanumGothic.js', 'export const NanumGothicBase64 = "' + b64 + '";');
console.log('완료! 크기:', b64.length);

File: frontend/convertFont.py
import base64 
 
with open('src/fonts/NanumGothic-Regular.ttf', 'rb') as f: 
    data = base64.b64encode(f.read()).decode() 
 
with open('src/fonts/NanumGothic.js', 'w') as f: 
    f.write('export const NanumGothicBase64 = "' + data + '";') 
 
print('완료! 크기:', len(data)) 
